neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -122,25 +169,100 @@ def create_experiment(self,
+            # points = [Point(
+            #     timestampMillis=int(value.ts * 1000.0),
+            #     x=value.x,
+            #     y=Y(numericValue=value.y.get('numeric_value'),
+            #         textValue=value.y.get('text_value'),
+            #         inputImageValue=value.y.get('image_value'))
+            # ) for value in channel_with_values.channel_values]
+            ch_values = [
+                alpha_operation.LogStrings.ValueType(
+                    value=value.y.get('text_value'),
+                    step=None,  # TODO: what is step?
+                    ts=int(value.ts),
+                )
+                for value in channel_with_values.channel_values
+            ]
+            ops.append(alpha_operation.LogStrings(
+                path=alpha_path_utils.parse_path(channel_with_values.channel_id),
+                values=ch_values,
+            ))
+
+        try:
+            # TODO: handle alpha exceptions
+            errors = self._alpha_backend.execute_operations(
+                experiment_uuid=uuid.UUID(experiment.internal_id),
+                operations=ops
+            )
+        except HTTPNotFound:
+            # pylint: disable=protected-access
+            raise ExperimentNotFound(
+                experiment_short_id=experiment.id, project_qualified_name=experiment._project.full_id)
+
     def get_system_channels(self, experiment):
-        return list()
+        params = {
+            'experimentId': experiment.internal_id,
+        }
+        try:
+            experiment = self.leaderboard_swagger_client.api.getExperimentAttributes(**params).response().result
+            return [
+                AlphaChannelWithLastValue(
+                    ch_id=attr.stringSeriesProperties.attributeName,
+                    ch_name=alpha_path_utils.parse_path(attr.stringSeriesProperties.attributeName)[-1],
+                    ch_type=attr.stringSeriesProperties.attributeType,
+                )
+                for attr in experiment.attributes
+                if (attr.type == AlphaAttributeType.STRING_SERIES.value
+                    and attr.name.startswith(MONITORING_ATTRIBUTE_SPACE))
+            ]
+        except HTTPNotFound:
+            # pylint: disable=protected-access
+            raise ExperimentNotFound(
+                experiment_short_id=experiment.id, project_qualified_name=experiment._project.full_id)
 
     def create_system_channel(self, experiment, name, channel_type):
-        return NonCallableMagicMock()
+        dummy_log_string = alpha_operation.LogStrings(
+            path=alpha_path_utils.parse_path(f'{MONITORING_ATTRIBUTE_SPACE}{name}'),
+            values=[],
+        )
+        # pylint: disable=unused-variable
+        errors = self._alpha_backend.execute_operations(
+            experiment_uuid=uuid.UUID(experiment.internal_id),
+            operations=[dummy_log_string],
+        )
+        system_channels = self.get_system_channels(experiment)
```

Comment:
Additional api call is required to construct, `AlphaChannelWithLastValue`.

Eventually we may assume that if there's no errors, operation completed, and create `AlphaChannelWithLastValue` without asking server for data.